### PR TITLE
Wanda's Python File

### PR DESCRIPTION
**Pull request by wanda-carlson**
This is a body message

### CODE REVIEW

**Comment by wanda-carlson** [comment]
This is the issue I've been having.
Screen Shot 2022-01-06 at 2 57 26 PM

**Comment by henry-richard** [comment]
I'll have a look.

File: src/wanda.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print("this is wanda")
```
Comment (henry-richard) [inline src/wanda.py:1]: This is ok

Comment (henry-richard) [inline src/wanda.py:1]: if only

**Review by henry-richard — CHANGES_REQUESTED** [review]
Coolios

**Review by henry-richard — APPROVED** [review]
Nice work @wanda-carlson !

**Pull request merged**